### PR TITLE
Feature indirect-ping handler

### LINKED ISSUE #74: [MessageHandler] implements handle, Indirect-Ping handler
implements MessageHandler handle, more specifically Indirect-Ping message handler

### PR DESCRIPTION
**Pull request by zeroFruit**
resolved: #74 

details:

write about indirect-ping handler w/ test cases

- [x] Test case

### CODE REVIEW

File: swim.go [hunk 1]
```diff
@@ -219,3 +224,68 @@ func (s *SWIM) handlePbk(piggyBack *pb.PiggyBack) {
 		s.pbkStore.Push(*piggyBack)
 	}
 }
+
+// handleIndirectPing receives IndirectPing message, so send the ping message
+// to target member, if successfully receives ACK message then send back again
+// ACK message to member who sent IndirectPing message to me.
+// If ping was not successful then send back NACK message
+func (s *SWIM) handleIndirectPing(msg pb.Message) {
+	seq := msg.Seq
```
Comment (junbeomlee) [inline swim.go:233]: I think we can change 
`func (s *SWIM) handleIndirectPing(msg pb.Message) {` to 
`func (s *SWIM) handleIndirectPing(msg pb. Message_IndirectPing) {`

**Comment by junbeomlee** [comment]
If we change 
```
switch msg.Payload.(type) {
	case *pb.Message_Ping:
		// handle ping
	case *pb.Message_Ack:
		// handle ack
	case *pb.Message_IndirectPing:
		s.handleIndirectPing(msg)
	default:
}
```
to  
```
switch msg := msg.Payload.(type) {
	case *pb.Message_Ping:
		// handle ping
	case *pb.Message_Ack:
		// handle ack
	case *pb.Message_IndirectPing:
		s.handleIndirectPing(msg)
	default:
}
```
then msg type can be *pb.Message_IndirectPing inside case *pb.Message_IndirectPing:

File: swim.go (hunk 1 above)
Comment (zeroFruit) [reply swim.go:233]: but if we change like that, we can't use seq string in message

**Comment by zeroFruit** [comment]
now piggyback data retrieved from `pbkstore`

**Comment by zeroFruit** [comment]
changed src, target variable name with mI, mJ

**Review by junbeomlee — APPROVED** [review]

**Pull request merged**